tsk/img/img_open.cpp

```diff
@@ -266,6 +290,24 @@ tsk_img_open(int num_img,
         break;
 #endif
 
+#if HAVE_LIBQCOW
+    case TSK_IMG_TYPE_QCOW_QCOW:
+        img_info = qcow_open(num_img, images, a_ssize);
+        break;
+#endif
+
+#if HAVE_LIBVHDI
```

Comment:
LIBVHDI and LIBVMDK cases are already handled above

Reply:
Sorry, can you be more specific? I don't see what you're referring to.